crates/integration/contracts/ExtCode.sol

```diff
@@ -2,6 +2,64 @@
 
 pragma solidity ^0.8;
 
+/* runner.json
+{
+    "actions": [
+        {
+            "Instantiate": {
+                "code": {
+                    "Solidity": {
+                        "contract": "ExtCode"
+                    }
+                }
+            }
+        },
+        {
+            "Call": {
+                "dest": {
+                    "Instantiated": 0
+                },
+                "data": "cbb20918"
+            }
+        },
+        {
+            "VerifyCall": {
+                "success": true,
+                "output": "10b3daf252aa843eae2b3d850da130d6a41ce1c23f33081c7fd8e48a0844add8"
```

Comment:
I fear this makes the test very brittle. Those JSON specs are intended for differential testing against EVM. But those code hashes are different than the corresponding EVM code bytecode hashes. And they change all the time, we don't want to manually update them each time the code hash of the fixture changes (I also want to avoid maintaining compiled fixtures).

I think the easiest way to write those tests without them breaking all the time is is to just write them iteratively. Basically just add the contract [like this](https://github.com/paritytech/revive/blob/main/crates/integration/src/cases.rs#L209C1-L216C66). Then you can easily implement the test manually somewhere [here](https://github.com/paritytech/revive/blob/main/crates/integration/src/tests.rs). The alternative is to make to extend the spec runner format and runner so that we can express in the spec JSON that the we want to compare against the bytecode hash of a contract. Which might be a lot more work than just writing this test iteratively :) And we have planned changes to the test spec format anyways. So just doing it iteratively would be fine for me.

Reply:
Yes I plan to do that